docs/source/ci-cd.md

```diff
@@ -187,6 +188,85 @@ pipelines:
 ```
 
 
+## Jenkins
+To set up `rover` for use with Jenkins, you'll first want to consider what type of Jenkins `agent` you'll be using in your pipelines. The below samples are showcasing a golang pipeline utilizing Docker, but can be extended to meet your specific needs. 
+
+### Pipelines using Distributed Builds via Node Agent
+If you're running a distributed build system using Jenkins's `node` agent type, you'll need to ensure that `rover` is installed on all machines as part of a baseline image, or via a setup script, and available via a `PATH` variable globally. 
+
+### Pipelines using Docker
+When using Docker containers as part of your build process, you'll need to be aware of a few additional things. Normally when installing, Rover adds the path of its executable to your `$PATH`. However, Jenkins doesn't persist the `$PATH` variable between runs of `sh`  `steps` as each `sh` block is run as its own process. This means that if you install Rover and try to run it in the next step, you get a `command not found: rover` error. This is functionally similar to [CircleCI note](#linux-jobs-using-the-curl-installer), however with a different solve.
+
+To avoid this issue, it is preferred to either: 
+- Use the script, but reference `rover` by full path (`$HOME/.rover/bin/rover`) 
+- Download the latest release via cURL and extract the binary via `curl -L https://github.com/apollographql/rover/releases/download/v0.7.0/rover-v0.7.0-x86_64-unknown-linux-gnu.tar.gz | tar --strip-components=1 -zxv` (for `0.7.0` on Linux x86)
+
+
+Secondly, you may run into permissions issues within Docker, and you can avoid some of this by creating a user to run the install and build process within. The below example Dockerfile shows how this can be accomplished with a specific Docker image for the Jenkins build pipeline. 
+
+```dockerfile
+FROM golang:1.18
+RUN useradd -m rover && echo "rover:rover" | chpasswd
+USER rover
+RUN curl -sSL https://rover.apollo.dev/nix/latest | sh
+```
+
+### Jenkinsfile Configuration
+
+Once you've installed `rover` appropriately, you can use the command within a `sh` step, as shown in the below example configuration. Since `rover` outputs logs via stderr and emits proper status codes, it will cause build errors if the `rover subgraph check` fails. 
```

Comment:
```suggestion
After you've installed Rover appropriately, you can execute the `rover` command within a `sh` step, as shown in the example configuration below. Because `rover` outputs logs via stderr and emits proper status codes, it generates build errors if the `rover subgraph check` command fails. 
```